examples/triangular_mesh_2D/elixir_euler_sbp_triangular_mesh.jl

```diff
@@ -0,0 +1,47 @@
+# !!! warning "Experimental features"
+
+using StartUpDG, StructArrays
+using Trixi, OrdinaryDiffEq
+
+rd = RefElemData(Tri(),SBP(), N=4)
```

Comment:
```suggestion
rd = RefElemData(Tri(), SBP(), N=4)
```
This one uses the SBP operators on simplices of Hicken et al., doesn't it?

Reply:
It depends on the options. Currently, it defaults to a set of degree SBP operators built from nodes with degree `2p-1` quadrature accuracy and `p+2` Lobatto nodes on the faces (from Ethan Kubatko). One can access Hicken's nodes via
```julia
rd = RefElemData(Tri(),SBP(), N, quadrature_strength=2*N) # Hicken nodes
rd = RefElemData(Tri(), SBP(), N, quadrature_strength=2*N-1, quad_rule_face=:Legendre) # Chen-Shu nodes
```

The kwarg API should probably be improved - I'm thinking about adding these as symbol fields in the `SBP()` type instead.

Reply:
Thanks, that sounds good :+1: